### PR TITLE
Added builds-view to show the list of builds for a given container id

### PR DESCRIPTION
**Pull request by munishchouhan**
This PR will add new view builds-view to show the list of builds for a given container id

### CODE REVIEW

**Comment by munishchouhan** [comment]
Tested locally:
Screenshot 2024-10-16 at 10 42 21
Screenshot 2024-10-16 at 10 42 06
Screenshot 2024-10-16 at 10 40 05

**Comment by pditommaso** [comment]
let's remove the digest, bit useless

File: src/main/groovy/io/seqera/wave/service/persistence/impl/SurrealPersistenceService.groovy [hunk 1]
```diff
@@ -182,6 +182,21 @@ class SurrealPersistenceService implements PersistenceService {
         return result
     }
 
+    @Override
+    List<WaveBuildRecord> allBuilds(String containerId) {
+        final query = """
+            select * 
+            from wave_build 
+            where buildId ~ '${containerId}${BuildRequest.SEP}'
+            order by buildId
```
Comment (pditommaso) [inline src/main/groovy/io/seqera/wave/service/persistence/impl/SurrealPersistenceService.groovy:191]: Better to order by time, the id could not be consist when passing from `_9` to `_10` and similar

**Comment by pditommaso** [comment]
For the table CSS styling, this can be a good start https://codepen.io/rglyall/pen/rVJJxL

File: src/main/groovy/io/seqera/wave/service/persistence/impl/SurrealPersistenceService.groovy (hunk 1 above)
Comment (munishchouhan) [reply src/main/groovy/io/seqera/wave/service/persistence/impl/SurrealPersistenceService.groovy]: done

**Comment by munishchouhan** [comment]
> let's remove the digest, bit useless

**Comment by munishchouhan** [comment]
is this better?
Screenshot 2024-10-16 at 11 14 54

**Comment by munishchouhan** [comment]
Screenshot 2024-10-16 at 11 35 55

**Comment by pditommaso** [comment]
Nice. Instead of exitStatus use [succeeded](https://github.com/seqeralabs/wave/blob/054aa1085496b5f7cff458da5f88583e058d9a46/src/main/groovy/io/seqera/wave/service/persistence/WaveBuildRecord.groovy#L61-L61) true|false

**Comment by pditommaso** [comment]
> I am trying to follow this table

but remove the vertical cell (internal) border, they are useless

**Comment by munishchouhan** [comment]
Screenshot 2024-10-16 at 11 44 58

File: src/main/groovy/io/seqera/wave/controller/ViewController.groovy [hunk 2]
```diff
@@ -171,7 +171,14 @@ class ViewController {
...
             bind.build_image = result.targetImage
-            bind.build_exit_status = result.exitStatus != null ? result.exitStatus : '-'
+            if( result.done() && result.succeeded() )
+                bind.build_status = "SUCCEEDED"
+            else if ( result.done() && !result.succeeded() )
+                bind.build_status = "FAILED"
+            else if (!result.done())
+                bind.build_status = "IN PROGRESS"
+            else
+                bind.build_status = "UNKNOWN"
```
Comment (pditommaso) [inline src/main/groovy/io/seqera/wave/controller/ViewController.groovy:181]: let's make a little function for this

Comment (munishchouhan) [reply src/main/groovy/io/seqera/wave/controller/ViewController.groovy]: done

Comment (pditommaso) [reply src/main/groovy/io/seqera/wave/controller/ViewController.groovy]: Thanks

**Comment by pditommaso** [comment]
@munishchouhan  Let's do the same for container scans, which have a similar pattern, in a separate PR

File: VERSION [hunk 3]
```diff
@@ -1 +1 @@
-1.13.1
+1.13.2-B1
```
Comment (pditommaso) [inline VERSION:1]:
```suggestion
1.13.1
```

**Review by pditommaso — APPROVED** [review]

**Pull request merged**